### PR TITLE
initial submission Barbarian Data System

### PR DESCRIPTION
**Pull request by grobbie**
Signed-off-by: Rob Gibbon <rob@barbarians.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/barbarian/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+**/charts/*.tgz
+**/*.lock
```
Comment (giacope) [inline incubator/barbarian/.gitignore]: There is no need for this `.gitignore`.

File: incubator/barbarian/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: Barbarian is a big data system founded on Apache Hadoop offering Hive, LLAP and Tez for fast SQL datawarehousing at scale.
+name: barbarian
+version: 0.1.2-badh0.1.0
+icon: https://barbarians.io/barbarians.png
+maintainers:
+- name: rob gibbon
```
Comment (giacope) [inline incubator/barbarian/Chart.yaml]: Please use your GitHub username.

File: incubator/barbarian/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: Barbarian is a big data system founded on Apache Hadoop offering Hive, LLAP and Tez for fast SQL datawarehousing at scale.
+name: barbarian
+version: 0.1.2-badh0.1.0
```
Comment (giacope) [inline incubator/barbarian/Chart.yaml]: Not 100% sure, but seems like this doesn't follow semantic versioning guidelines (which is a requirement).

File: incubator/barbarian/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```
Comment (giacope) [inline incubator/barbarian/Chart.yaml:1]: Please add an `appVersion` field

File: incubator/barbarian/LICENSE [hunk 5]
```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```
Comment (giacope) [inline incubator/barbarian/LICENSE]: `helm/charts` is already under Apache `2.0`, so I don't think there is a need to include the license again.

File: incubator/barbarian/barbarian/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```
Comment (giacope) [inline incubator/barbarian/barbarian/Chart.yaml]: Same here.

File: incubator/barbarian/barbarian/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+description: Barbarian is a big data system founded on Apache Hadoop offering Hive, LLAP and Tez for fast SQL datawarehousing at scale.
+name: barbarian
+version: 0.1.2-badh0.1.0
```
Comment (giacope) [inline incubator/barbarian/barbarian/Chart.yaml]: Same here

File: incubator/barbarian/barbarian/Chart.yaml [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+icon: https://barbarians.io/barbarians.png
+maintainers:
+- name: rob gibbon
```
Comment (giacope) [inline incubator/barbarian/barbarian/Chart.yaml]: Same here.

File: incubator/barbarian/barbarian/templates/hive-hms-service.yaml [hunk 9]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.hive_hms.service.name }}
+  labels:
```
Comment (giacope) [inline incubator/barbarian/barbarian/templates/hive-hms-service.yaml]: Missing standard labels.

Comment (grobbie) [reply incubator/barbarian/barbarian/templates/hive-hms-service.yaml]: added standard labels throughout

File: incubator/barbarian/barbarian/Chart.yaml (hunk 8 above)
Comment (grobbie) [reply incubator/barbarian/barbarian/Chart.yaml]: changed to use github username, corrected indentation

File: incubator/barbarian/barbarian/Chart.yaml (hunk 7 above)
Comment (grobbie) [reply incubator/barbarian/barbarian/Chart.yaml]: adapted version to semver2 spec

File: incubator/barbarian/Chart.yaml (hunk 4 above)
Comment (grobbie) [reply incubator/barbarian/Chart.yaml:1]: done

**Comment by grobbie** [comment]
It's all good from my perspective. Please re-review and feedback any concerns. Thanks!

**Comment by prydonius** [comment]
@grobbie any particular reason to put this into incubator? It could go directly into stable

**Comment by grobbie** [comment]
I chose incubator because the setup is still quite fresh and will continue to develop as a product for a while. Stable would be really cool, but I think I'll keep it in incubator for now if that's ok?

**Pull request closed without merging**